pyproject.toml

```diff
@@ -21,6 +21,7 @@ classifiers = [
   "Programming Language :: Python :: 3.9",
   "Programming Language :: Python :: 3.10",
   "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
```

Comment:
Here `"Programming Language :: Python :: 3.7",` should also be removed, given the title of the PR.